ethereum/permissioning/src/main/java/org/hyperledger/besu/ethereum/permissioning/TransactionSmartContractPermissioningController.java

```diff
@@ -171,7 +171,7 @@ public static Boolean checkTransactionResult(final Bytes result) {
     // 0 is false
     if (result.equals(FALSE_RESPONSE)) {
       return false;
-      // 32 bytes of 1's is true
+      // A 32 byte versios of 1 is true
```

Comment:
Looks like a typo in this new comment.  Given we have constants which make it clear what the magic values mean I'm not sure the comments are actually helpful anyway...

Reply:
Yes there is a typo. I will fix it. 

Comments that are misleading are **really** not helpful - this was motivated because I was trying to understand how Besu managed this particular issue, so lost a bunch of time noticing, checking, and understanding the discrepancy. 
("The code is the documentation" isn't so great either, which is why the comments are there in the first place).

(Also, sorry for the typo and the time it burns. I also saw this as an exercise in figuring out the workflow in practice, that wouldn't require complex review :( )

Reply:
No problem @chaals, and thank you for your contribution. If you were able to spend the time, would you please look for tests covering this particular class? Do they cover well the use case and can they participate in documentation?

Reply:
I did look at the tests (first - and then I checked the code to make sure I had understood correctly...), and they do seem to cover the use case...

The behaviour for this EEA required feature is actually unspecified - I got here because I am trying to work out what makes sense to specify.